adapters/tlsio_wolfssl.c

```diff
@@ -103,6 +103,23 @@ static void* tlsio_wolfssl_CloneOption(const char* name, const void* value)
                 /*return as is*/
             }
         }
+        #ifdef INVALID_DEVID
+        else if(strcmp(name, OPTION_WOLFSSL_SET_DEVICE_ID) == 0 )
+        {
+             int* value_clone; 
+  
+             if ((value_clone = (int*)malloc(sizeof(int))) == NULL) 
+             { 
+                 LogError("unable to clone device id option"); 
+             } 
+             else 
+             { 
+                 *value_clone = *value; 
```

Comment:
This is not building because cannot dereference a void pointer.  Please cast to int pointer first:

```
*value_clone = *(int*)value; 
```

Reply:
resolved